crates/bones_input/src/time/mod.rs

```diff
@@ -0,0 +1,299 @@
+//! Time functionality for the Bones framework.
```

Comment:
Is this directly "stolen" from Bevy? If it is, maybe we should just put a comment at the top of the module with the permalink to where the code was taken from and a note that it's licensed under either MIT or Apache 2.0.

Maybe that's a little paranoid, and we do already depend on Bevy anyway, but it doesn't hurt anything.